source/retryable-reads/tests/README.rst

```diff
@@ -232,6 +232,56 @@ This test requires MongoDB 4.2.9+ for ``blockConnection`` support in the failpoi
 
 9. Disable the failpoint.
 
+Retrying Reads in a Sharded Cluster
+===================================
+
+These tests will be used to ensure drivers properly retry reads on a different
+mongos.
+
+Retryable Reads Are Retried on a Different Mongos if One Available
+------------------------------------------------------------------
+
+This test MUST be executed against a sharded cluster that has at least two
+mongos instances.
+
+1. Ensure that a test is run against a sharded cluster that has at least two
+   mongos-es. If there are more than two mongos-es in the cluster, pick two to
+   test against.
+
+2. Create a client per mongos using the direct connection, and configure fail
+   points on each of the picked mongos-es, so that each mongos raises
+   a retryable error once.
```

Comment:
I suggest noting not to set a failpoint error that is transient, according to the [specifications](https://github.com/mongodb/specifications/blob/master/source/retryable-reads/retryable-reads.rst#why-are-read-operations-only-retried-once-by-default):

>  In the case of a transient error, the driver will mark the server as "unknown"

This could result in false positives. For example, suppose we have two servers: s1, s2.  If s1 fails on the initial attempt, it's type will be set to unknown and is therefore no longer a suitable candidate for server selection (see [here](https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#topology-type-unknown)). Thus s2 MUST be selected on the retry and would "fail". In this case, this prose test would pass with or without implementing the mongos retry logic.